routes/api.js

```diff
@@ -0,0 +1,112 @@
+var express = require('express');
+var router = express.Router();
+var userData = require('../userDetails.json');
+
+
+router.get('/posts',function(req,res,next){
+  if(req.cookies.auth_token){
+    next();
+  }
+  else{
+    res.send('Ur Not Authorized');
+  }
+},function(req,res){
+  const cookie = req.cookies;
+  userData.forEach(user => {
+    if(user.userName===cookie.username){
+      res.send(user.posts);
+    }
+  });
+})
+router.post('/posts/new',function(req,res){
+  const cookie = req.cookies;
+  const newStory = {
+    "story" : req.body.story
+  };
+  userData.forEach(user => {
+    if(user.userName===cookie.username){
```

Comment:
doing a check here is not recommended. all auth related you must be doing in a middleware.

Reply:
Resolved